.github/workflows/main.yml

```diff
@@ -2,23 +2,32 @@ name: build
 on: [push, pull_request]
 jobs:
   validate:
-    name: Validate 
+    name: Validate
     runs-on: ubuntu-latest
+    env:
+      GOPATH: ${{ github.workspace }}/go
+    defaults:
+      run:
+        working-directory: ${{ env.GOPATH }}/src/github.com/kubeflow/mpi-operator
```

Comment:
We must set a proper working directory to avoid the following error:

> Generating V2 OpenAPI specification ...
> API rule violation: list_type_missing,github.com/kubeflow/common/pkg/apis/common/v1,JobStatus,Conditions
> Generating V2 swagger file ...
> # command-line-arguments
> Error: hack/python-sdk/main.go:48:23: undefined: v2beta1.GetOpenAPIDefinitions
> make: *** [Makefile:80: generate] Error 2
> Error: Process completed with exit code 2.

https://github.com/kubeflow/mpi-operator/actions/runs/4047152597/jobs/6960826644#step:7:67